docs/aws_s3_compat.md

```diff
@@ -170,6 +170,12 @@ See also `GetObject` and other method parameters.
 | 🟢 | ListBuckets          |           |
 | 🔵 | PutPublicAccessBlock |           |
 
+* The `CreateBucket` method allows you to select a placement policy using the `LocationConstraint` parameter in the AWS CLI. The policy name should be passed as a value.
```

Comment:
`s/The//`, some "Extensions" section would be nice to have.

Reply:
Removed _The_